mongodb:
  host: 'localhost'
  port: 62345
vision-project:
  controllers:
    is_debug: False
  is_run_demo_interaction: True
  seconds_until_interaction_timeout: 45
  resources:
    path:
      demo: '/root/catkin_ws/src/vision-project/src/ros_vision_interaction/resources/demo-interaction/'
      deployment: '/root/catkin_ws/src/vision-project/src/ros_vision_interaction/resources/deployment/'
